package.json

```
@@ -29,5 +30,10 @@
     "eslint": "7.32.0",
     "ts-node": "10.2.1",
     "typescript": "4.4.3"
+  },
+  "dependencies": {
+    "D": "^1.0.0",
+    "express": "^4.17.1",
+    "nodemon": "^2.0.15"
```

Esto no debería ir en devDependencias?

```suggestion
    "nodemon": "2.0.15"
```

---

Ya lo cambiamos arriba y le corregimos la versión 😎